.github/workflows/release-umee.yml

```diff
@@ -28,26 +28,32 @@ jobs:
       - name: Set Env
         run: echo "TM_VERSION=$(go list -m github.com/cometbft/cometbft | sed 's:.* ::')" >> $GITHUB_ENV
 
+      - name: Set Wasmvm Env
+        run: echo "COSMWASM_VERSION=$(go list -m github.com/CosmWasm/wasmvm | sed 's/.* //')" >> $GITHUB_ENV
+
+
+
       # useful to test builds. However will require to add "push" rule to the "on" section
       - name: generate and update swagger docs
         run: |
           make proto-swagger-gen
           make proto-update-swagger-docs
 
       - name: goreleaser test-build
-        uses: goreleaser/goreleaser-action@v6
+        uses: goreleaser/goreleaser-action@v5
```

Comment:
I had to downgrade the goreleaser version because of this [bug](https://github.com/goreleaser/goreleaser/issues/4914). Logs of the failed run using the latest version with the same bug can be found on my fork [here](https://github.com/kaustubhkapatral/umee/actions/runs/9643703524/job/26594246179#step:8:22)